Windows npm start error

I ran npm install, and then npm start, and got this error:

```
npm start

> scratch-gui@0.1.0 start C:\Users\johnr\Documents\scratch-gui
> webpack-dev-server --port $npm_package_config_port --content-base=./build

events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: listen EACCES $npm_package_config_port
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at Server._listen2 (net.js:1249:19)
    at listen (net.js:1298:10)
    at Server.listen (net.js:1382:7)
    at Server.listen (C:\Users\johnr\Documents\scratch-gui\node_modules\webpack-dev-server\lib\Server.js:321:27)
    at Object.<anonymous> (C:\Users\johnr\Documents\scratch-gui\node_modules\webpack-dev-server\bin\webpack-dev-server.js:183:37)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
    at Module.runMain (module.js:607:10)
    at run (bootstrap_node.js:382:7)
    at startup (bootstrap_node.js:137:9)
    at bootstrap_node.js:497:3

npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\johnr\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "start"
npm ERR! node v7.0.0
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE
npm ERR! scratch-gui@0.1.0 start: `webpack-dev-server --port $npm_package_config_port --content-base=./build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the scratch-gui@0.1.0 start script 'webpack-dev-server --port $npm_package_config_port --content-base=./build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the scratch-gui package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     webpack-dev-server --port $npm_package_config_port --content-base=./build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs scratch-gui
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls scratch-gui
npm ERR! There is likely additional logging output above.
```

--- a/package.json
+++ b/package.json
@@ -3,9 +3,6 @@
   "version": "0.1.0",
   "description": "GraphicaL User Interface for creating and running Scratch 3.0 projects",
   "main": "./src/index.js",
-  "config": {
-    "port": 8601
-  },
   "scripts": {
     "build": "npm run clean && webpack --progress --colors --bail",
     "clean": "rm -rf ./build && mkdir -p build",
@@ -14,7 +11,7 @@
     "postmerge": "opt --in postmerge --exec 'npm install'",
     "postrewrite": "opt --in postrewrite --exec 'npm install'",
     "prepush": "opt --in prepush --exec 'npm run lint'",
-    "start": "webpack-dev-server --port $npm_package_config_port --content-base=./build",
+    "start": "webpack-dev-server",
     "test": "npm run lint && npm run build"
   },
   "author": "Massachusetts Institute of Technology",

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -6,6 +6,11 @@ var HtmlWebpackPlugin = require('html-webpack-plugin');
 var webpack = require('webpack');
 
 module.exports = {
+    devServer: {
+        contentBase: path.resolve(__dirname, 'build'),
+        host: '0.0.0.0',
+        port: process.env.PORT || 8601
+    },
     entry: {
         lib: ['react', 'react-dom'],
         gui: './src/index.jsx'

